modules/cudf/src/data_frame.ts

```diff
@@ -345,6 +432,31 @@ export class DataFrame<T extends TypeMap = any> {
       {} as SeriesMap<{[P in keyof T]: R}>));
   }
 
+  /**
+   * Concat DataFrame(s) to the end of the caller, returning a new DataFrame.
+   *
+   * @param dfs The DataFrame(s) to concat to the end of the caller.
+   *
+   * @example
+   * ```typescript
+   * import {DataFrame, Series} from '@rapidsai/cudf';
+   * const df = new DataFrame({
+   *   a: Series.new([1, 2, 3, 4]),
+   * });
+   *
+   * const df2 = new DataFrame({
+   *   b: Series.new([5, 6, 7, 8]),
+   * });
+   *
+   * df.concat(df2);
+   * // return {
+   * //    a: [1, 2, 3, 4, null, null, null, null],
+   * //    b: [null, null, null, null, 5, 6, 7, 8],
```

Comment:
Oh, hrm , this is actually the case I meant with the comment about pandas above. It you try to concat tow df's that have no columns in common at all, it raises an error. Maybe we do want to do this for node-rapids, though

Reply:
Reminder we shouldn't target Pandas behavior, except incidentally due to targeting cuDF Python.

Reply:
so I'm assuming keeping it the same is fine then, this is how `cuDF` handles it when attempting it on my end.

Reply:
Definitely, I compared to pandas because I happened to be on my laptop at the time, but I should have waited to comment until I could have checked cudf on the workstation